src/themes/masschallenge/elements/container.overrides

```diff
@@ -0,0 +1,24 @@
+/*******************************
+         Theme Overrides
+*******************************/
+
+@media screen and (min-width: 768px) and (max-width: 1440px) {
+.container{
+    display: grid;
+    grid-column: 2;
+    grid-template-columns: 220px 960px;
```

Comment:
I believe most of these values could be replaced with variables.

Reply:
Let me make them variables. Thanks.

Reply:
I have made the changes with the variables.

Reply:
@hfreund Yes we did and I made the relevant changes to the sizing of the second column.